src/containers/shared/test/txDetails.test.js

```diff
@@ -22,7 +22,7 @@ import TxDetails from '../components/TxDetails';
 import summarize from '../../../../server/lib/txSummary';
 
 describe('TxDetails', () => {
-  const createWrapper = (tx, width = 1200) =>
+  const createWrapper = tx =>
```

Comment:
Was width no longer being used.

Reply:
The variable wasn't being used in the method, so I took it out.